### DISCUSSION TITLE
UTM will not use wired ethernet on macOS

### DISCUSSION THREAD

**Discussion by wingcomm** [utmapp/UTM#4710] [UTM Help (macOS)] [answered]
**Describe the issue**  
UTM will not use wired ethernet adapter on macOS. Will only bridge traffic onto the Wi-Fi Adapter. I tested both Shared and Bridged modes and experienced the same behavior. I can only use the Wi-Fi adapter.

**Configuration**  
* UTM Version: 4.0.9 (68)
* macOS Version: 12.6.1
* Mac Chip (Intel, M1, ...): M1 Max

**Debug log**  
[debug.log](https://github.com/utmapp/UTM/files/9971955/debug.log)

**Comment by conath** [comment, marked as answer]
You might be able to change the network that is **shared** by re-arranging the networks in System Preferences to have Ethernet at the top of the list.

When using bridged mode, you can change the interface that is used for the bridge in the UTM VM configuration Network section. Enter the interface ID where it says "Bridged Interface – en0". Wi-Fi is always en0 on Macs, so your ethernet could be en1 (check in Terminal).

**Comment by wingcomm** [comment]
Thank you @conath, but this is not a workable solution. I need to share the network kernel on macOS to maintain a single VPN connection and access other connected network resources as I move between wireless and wired networks on my laptop. This works in the Shared Network mode on Parallels and Fusion. There's no reason why it can't work here in UTM. @osy can you unmark this as the solution?

**Reply by kExU9853** [reply]
Hi folk, I just met the exact same problems. Are there any solutions now?
